subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/mutator/ClearArtifactTransformCacheWithoutInstrumentedJarsMutator.groovy

```diff
@@ -82,4 +109,20 @@ class ClearArtifactTransformCacheWithoutInstrumentedJarsMutator extends ClearArt
             }
         })
     }
+
+    static BuildMutator create(File gradleUserHome, CleanupSchedule schedule) {
+        def clearOldTransformsDir = new ClearArtifactTransformCacheWithoutInstrumentedJarsMutator(
+            new File(gradleUserHome, "caches"),
+            schedule,
+            "transforms-")
+
+        def clearNewTransformsDir = new ClearArtifactTransformCacheWithoutInstrumentedJarsMutator(
+            new File(gradleUserHome, "caches/" + GradleVersion.current().version),
```

Comment:
🤔 Oh I see this is more complicate now. Sorry for troubles with that.
I think if we do `GradleVersion.current().version` we will get a version from Gradle that invokes the test. So I think we will need to get versions from  `invocationSettings.getVersions()`

Reply:
If you won't be able to verify that today I can also do it during a week and make sure we merge that before branching.

Reply:
I pushed 4f1daf710733b3f099b8fea74899b0723669b7c2 which should do the trick.